persistence/src/main/java/com/epam/esm/pool/DataSource.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm.pool;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+/**
+ * Connection pool
+ *
+ * @see com.zaxxer.hikari.HikariDataSource
+ */
+public class DataSource {
```

Comment:
@belikandrey Why do we need a separate class for the data source? Could we combine this class with `PersistenceSpringConfig`?

Reply:
Sorry. Now is up to date version